GDEdit Web/GDE.Web/Data/Helper.cs

```diff
@@ -2,8 +2,7 @@
 {
     public static class Helper
     {
-        public static int SectionToHueMap(LinkItems section)
-            => section switch
+        public static int SectionToHueMap(LinkItems section) => section switch
             {
```

Comment:
Move brackets 4 spaces to the left too, since the expression lies on that indentation after moving the switch on the above line.